src/libpriv/rpmostree-core.c

```diff
@@ -4018,12 +4025,13 @@ rpmostree_context_assemble (RpmOstreeContext      *self,
           DnfPackage *pkg = (void*)rpmteKey (te);
           g_assert (pkg);
 
+          rpmostree_output_set_sub_message (dnf_package_get_name (pkg));
           if (!run_script_sync (self, tmprootfs_dfd, &var_lib_rpm_statedir,
                                 pkg, RPMOSTREE_SCRIPT_PREIN,
                                 &n_pre_scripts_run, cancellable, error))
             return FALSE;
         }
-      rpmostree_output_task_done_msg (&task, "%u done", n_pre_scripts_run);
+      rpmostree_output_progress_end_msg (&task, "%u done", n_pre_scripts_run);
```

Comment:
FWIW, I don't mind the tab shifting to reflow this. We'll be reading it much more often than we'll be looking at it in `git show/blame` :)